src/fty_info_server.c

```diff
@@ -142,7 +142,11 @@ fty_info_server (zsock_t *pipe, void *args)
                  }
                  if (streq(command, "VERSION")) {
                     zmsg_t *reply = zmsg_new ();
+                    // are we on RC?
                     char *version = s_readall ("/etc/bios-release.json");
+                    if (version == NULL)
+                        // maybe we are on the container
+                        version = s_readall ("/usr/share/bios-web/image-version.txt");
```

Comment:
This is just a proof-of-concept before I have the opportunity to discuss the interface with the Convergence team.